scripts/zones/Ceizak_Battlegrounds/mobs/Supernal_Chapuli.lua

```diff
@@ -0,0 +1,61 @@
+-----------------------------------
+-- Area: Ceizak Battlegrounds
+-- NM: Supernal Chapuli
+-----------------------------------
```

Comment:
This will also need comments with commands for `!pos` (take you to the NM's spawn point), and `!additem` or some sort of `!exec` to force-pop the NM, for easier testing